docs/es/newbs_best_practices.md

```diff
@@ -0,0 +1,159 @@
+# Mejores prácticas
+
+## O, "Cómo aprendí a dejar de preocuparme y amarle a Git."
+
+Este documento procura instruir a los novatos en las mejores prácticas para tener una experiencia más fácil en contribuir a QMK. Te guiaremos por el proceso de contribuir a QMK, explicando algunas maneras de hacerlo más fácilmente, y luego romperemos algunas cosas para enseñarte cómo arreglarlas.
+
+En este documento suponemos un par de cosas:
+
+1. Tienes una cuenta de Github, y has hecho un [fork del repo qmk_firmware](getting_started_github.md) en tu cuenta.
+2. Has [configurado tu entorno de desarrollo](newbs_getting_started.md?id=environment-setup).
+
+
+## La rama master de tu fork: Actualizar a menudo, nunca commit
+
+Se recomienda que para desarrollo con QMK, lo que sea que estés haciendo, mantener tu rama `master` actualizada, pero **nunca** commit en ella. Mejor, haz todos tus cambios en una rama de desarrollo y manda pull requests de tus ramas mientras programas.
+
+Para evitar los conflictos de merge &mdash; cuando dos o más usuarios han editado la misma parte de un archivo al mismo tiempo &mdash; mantén tu rama `master` actualizada, y empieza desarrollo nuevo creando una nueva rama.
+
+### Actualizando tu rama master
+
+Para mantener tu rama `master` actualizada, se recomienda agregar el repository ("repo") de Firmware QMK como un repo remoto en git. Para hacer esto, abre tu interfaz de línea de mandatos y ingresa:
```

Comment:
```suggestion
Para mantener tu rama `master` actualizada, se recomienda agregar el repository ("repo") de Firmware QMK como un repo remoto en git. Para hacer esto, abre tu interfaz de línea de comandos e ingresa:
```
I had never seen command line as `línea de mandatos` I always see that as `línea de comandos` (I always set my software to `es-es` so I cannot confirm it is like that on other variants of Spanish.